languages/python/generic/Python_to_generic.ml

```diff
@@ -560,6 +560,11 @@ and fieldstmt x =
 and stmt_aux env x =
   match x with
   | FunctionDef (t, v1, v2, v3, v4, v5) ->
+      let fkind =
```

Comment:
Given these changes I find the PR title a bit unrelated (and way too long... you can elaborate in the PR description).

Reply:
This PR basically distinguishes functions versus methods in Python-to-Generic translation. I think the PR title your reflect just that, then you can always say what you want to do that or link to another related PR or ticket.